api/openapi.gen.json

```diff
@@ -71,6 +123,32 @@
         },
         "type": "object"
       },
+      "v1.ReservationResponse": {
+        "properties": {
+          "created_at": {
+            "format": "date-time",
+            "type": "string"
+          },
+          "finished_at": {
+            "format": "date-time",
+            "type": "string"
+          },
+          "id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "provider": {
+            "type": "integer"
+          },
+          "status": {
+            "type": "string"
+          },
+          "success": {
+            "type": "boolean"
+          }
+        },
+        "type": "object"
+      },
```

Comment:
Finally, this looks good :-) One idea: maybe we could name the schema name as v1.GenericReservationResponse to really differentiate it. But that is just an idea.